e2e/whisper/whisper_mailbox_test.go

```diff
@@ -83,16 +83,17 @@ func (s *WhisperMailboxSuite) TestRequestMessageFromMailboxAsync() {
 	messages := s.getMessagesByMessageFilterID(rpcClient, messageFilterID)
 	s.Require().Equal(0, len(messages))
 
-	//Post message
+	//Post message matching with filter (key and token)
 	s.postMessageToPrivate(rpcClient, pubkey.String(), topic.String(), hexutil.Encode([]byte("Hello world!")))
 
-	//There are no messages, because it's a sender filter
+	//get message to make sure that it will come from the mailbox later
```

Comment:
Nitpick: it would be nice to have `G` capitalized.

Reply:
Sure, done.